packages/mobile/src/harmony-native/foundations/color/Color.stories.tsx

```diff
@@ -122,7 +121,7 @@ function ColorSwatch(props: ColorSwatchProps) {
 
 type ColorRowProps = {
   description: string
-  colors: PrimitiveColors[keyof PrimitiveColors]
+  colors: any
```

Comment:
👀

Reply:
lol ^^ i felt it was not worth defining this type as this page is subject to a ton of change anway :)